src/main/java/uk/gov/dhsc/htbhf/productionstub/controller/v2/DWPBenefitControllerV2.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.dhsc.htbhf.productionstub.controller.v2;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.dhsc.htbhf.productionstub.model.v2.*;
+
+import static uk.gov.dhsc.htbhf.productionstub.model.v2.QualifyingBenefits.NOT_SET;
+
+@RestController
+@RequestMapping("/v2/dwp/benefits")
+@Slf4j
+@AllArgsConstructor
+public class DWPBenefitControllerV2 {
+
+    @GetMapping
+    public IdentityAndEligibilityResponse determineEligibility() {
+        IdentityAndEligibilityResponse response = IdentityAndEligibilityResponse.builder()
+                .addressLine1Match(VerificationOutcome.NOT_MATCHED)
+                .deathVerificationFlag(DeathVerificationFlag.N_A)
+                .eligibilityStatus(EligibilityOutcome.NOT_CONFIRMED)
+                .emailAddressMatch(VerificationOutcome.NOT_MATCHED)
+                .mobilePhoneMatch(VerificationOutcome.NOT_MATCHED)
+                .postcodeMatch(VerificationOutcome.NOT_MATCHED)
+                .pregnantChildDOBMatch(VerificationOutcome.NOT_MATCHED)
+                .qualifyingBenefits(NOT_SET)
+                .identityStatus(IdentityOutcome.NOT_MATCHED)
```

Comment:
This response isn't correct if the IdentityStatus is NOT_MATCHED, it should be:
```
        return IdentityAndEligibilityResponse.builder()
                .identityStatus(IdentityOutcome.NOT_MATCHED)
                .eligibilityStatus(EligibilityOutcome.NOT_SET)
                .qualifyingBenefits(QualifyingBenefits.NOT_SET)
                .mobilePhoneMatch(VerificationOutcome.NOT_SET)
                .emailAddressMatch(VerificationOutcome.NOT_SET)
                .addressLine1Match(VerificationOutcome.NOT_SET)
                .postcodeMatch(VerificationOutcome.NOT_SET)
                .pregnantChildDOBMatch(VerificationOutcome.NOT_SET)
                .householdIdentifier(NO_HOUSEHOLD_IDENTIFIER_PROVIDED)
                .dobOfChildrenUnder4(emptyList())
                .deathVerificationFlag(DeathVerificationFlag.N_A)
                .build();
```

Reply:
Updated the response.